web/src/components/profile/ProfileNewJerseyAddress.test.tsx

```diff
@@ -30,6 +45,14 @@ describe("<ProfileNewJerseyAddress  />", () => {
     const address = generateAddress({
       addressLine1: "1111 Home Alone",
     });
+    useMockBusiness(
+      generateBusiness({
+        formationData: generateFormationData({
+          completedFilingPayment: false,
+          formationFormData: generateFormationFormData({ ...address }),
+        }),
+      })
+    );
     renderComponent({ address });
```

Comment:
based on how the a/c is written, I think the component from all the tests in this file should have an empty address, in other words, address in not included as an argument. this is b/c the address in formationFormData will be used

Reply:
<WithStatefulAddressData
        initialData={ emptyAddressData} // removed address

Reply:
We can refactor this on refactor Friday.

Reply:
Pivotal Tracker: [Refactor ProfileNewJerseyAddress Tests With An Empty Address](https://www.pivotaltracker.com/n/projects/2573455/stories/188108828)

Reply:
I'm ok with this being a refactor ticket for now, but going forward I think I'd like to quickly address feedback like this in the PR. I don't know that we need to create additional work here.

Reply:
Done!